samples/calculator/androidApp/build.gradle

```diff
@@ -38,7 +34,7 @@ repositories {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-//    implementation "org.jetbrains.kotlin.konan.samples:arithmetic-parser"
+    expectedBy project(':parser')
```

Comment:
Isn't it a misuse of MPP? I mean, maybe we should convert the parser library itself to multiplatform project, and depend on it here.